pkg/node/state_fsm/fsm_common.go

```diff
@@ -182,3 +157,7 @@ func validateEventArgs(event stateless.Trigger, args ...interface{}) {
 		}
 	}
 }
+
+func clearSyncPeer(info BaseInfo) {
+	info.syncPeer.SetPeer(nil)
+}
```

Comment:
I think `network.SyncPeer` should have a method (for example, `Reset`) with this functionality.

Reply:
Fixed